lib/Raven/Stacktrace.php

```diff
@@ -1,166 +1,344 @@
 <?php
+
+/*
+ * This file is part of Raven.
+ *
+ * (c) Sentry Team
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
 namespace Raven;
 
 /**
- * Small helper class to inspect the stacktrace
+ * This class contains all the information about an error stacktrace.
  *
- * @package raven
+ * @author Stefano Arlandini <sarlandini@alice.it>
  */
-class Stacktrace
+class Stacktrace implements \JsonSerializable
```

Comment:
It's available since PHP 5.4.0 and (quoting from the docs) objects implementing [this interface](http://php.net/manual/en/class.jsonserializable.php) can customize their JSON representation when encoded with `json_encode()`